ads/_config.js

```diff
@@ -642,6 +642,10 @@ export const adConfig = {
 
   pubexchange: {},
 
+  pubguru: {
+    renderStartImplemented: true,
```

Comment:
Could you confirm that the `renderStart` API is rolled out? When I try to test with your example, I couldn't receive the `render-start` message. Note that AMP runtime won't display an ad before that message is received or timeout.

Reply:
Sorry the issue is that i was only calling renderStart when we had a winning ad, but forgot to implement the test version, it is updated. Please check again. Thanks